src/storage_pool.cc

```diff
@@ -116,7 +117,16 @@ NAN_METHOD(StoragePool::LookupByUUID)
 NAN_METHOD(StoragePool::LookupByVolume)
 {
   Nan::HandleScope scope;
-  return;
+  auto hv = ObjectWrap::Unwrap<Hypervisor>(info.This())->handle_;
+  auto volume = ObjectWrap::Unwrap<StorageVolume>(info[0]->ToObject())->handle_;
+  
+  Worker::Queue(info[1], [=](Worker::SetOnFinishedHandler onFinished) {
```

Comment:
I guess that confusion about `info` not being `args` should be attributed/blamed on Nan. :-) I guess there are good reasons for it not being `args` too since it also has `This()`. I wonder if we can do something better about handling, checking and validating arguments in general so that it's more declarative.